IoProtocol.h

```diff
@@ -134,18 +134,11 @@ struct      // IOP_TYP1_CMD_DATA
 SDioRequest;
 
 // Request for Digital IO modules: output with variable number of pwm values
-typedef
-#include <COMP_packBegin.h>
-struct      // IOP_TYP1_CMD_DATA2
```

Comment:
It would probably be good to retain the `// IOP_TYP1_CMD_DATA2` comment so that a casual readers knows `pwm_data` is a payload of that type.